moped-editor/src/queries/staff.js

```diff
@@ -28,10 +28,7 @@ export const GET_USER = gql`
       staff_uuid
       title
       user_id
-      moped_workgroup {
-        workgroup_id
-        workgroup_name
-      }
+      workgroup_id
```

Comment:
We can request the `workgroup_id` here so that we don't have to get it from the `moped_workgroup` part of the return object which required special handling to set it in the form that I mentioned above.